docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,222 @@
+
+```ssh
+export NETWORK_NAME=rinkeby
+export GAS_PRICE_GWEI=1
+export MASTER_SAFE=<your master safe>
+export PK=<the private key of the proposer account>
+```
+
+```ssh
+cd dex-liquidity-provision
+yarn install
+yarn compile
+yarn run networks-inject
+npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=7 --lowestLimit=150 --highestLimit=260 --currentPrice=200 --masterSafe=$MASTER_SAFE --depositBaseToken=5 --depositQuoteToken=1000 --numBrackets=20 --network=$NETWORK_NAME
+```
+
+The script will first make some plausibility checks as the mentioned price check and that the boundaries - highestLimit and lowerLimit - are reasonably set for the current price.
+
+It will then create a transaction deploying new safes - one new safe for each bracket. All new deployed Safe addresses will be printed on the screen.
+
+Afterwards, the transaction containing all the orders is built.
+All the order placements are bundled into one transaction, appearing first in the Gnosis Safe Interface.
+Before you sign the transaction, make sure to check the correctness of the displayed orders from the script's output.
+
+The transaction transferring the DAI and WETH into the brackets and depositing, on their behalf, into the GP exchange contract is built in parallel.
+This is the second transaction sent to the interface.
+
+If everything goes smoothly, you should see the two transaction initiations confirmed like that:
+
+```ssh
+
+==> Sending the order placing transaction to gnosis-safe interface.
+    Attention: This transaction MUST be executed first!
+
+Signing and posting multi-send transaction 0x007ecc58f9dc222c26215b260abc363e0308dfec539d7eeabb8a2c9f09397157 from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+
+==> Sending the funds transferring transaction, please execute this transaction second
+
+Signing and posting multi-send transaction 0x09cb78a5a49f10305a2f108d45e8fa059e4231ebf219315c56409d67419d65fb from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+```
+
+Now, you should just follow the link and sign & execute the transaction with the lower transaction-ID first. It should look like this:
+
+![Read tokenAddressToIdMap](assets/tutorial_bracket_strategy_signing.png)
+
+For executing, just press the button _confirm_.
+
+Once this first transaction is mined, execute the second one.
+
+Congrats! You just provided liquidity and have a chance to earn some passive income if the prices are moving and returning back to your initial `currentPrice`.
+
+## Withdrawing liquidity
```

Comment:
Note: include the changes from [this PR](https://github.com/gnosis/dex-liquidity-provision/pull/279) when it will be merged.